docs/api/vast-tracker.md

```diff
@@ -136,7 +136,28 @@ Macros will be replaced and the skip tracking URL will be called with the follow
 
 ## Public Methods 💚 <a name="methods"></a>
 
-### errorWithCode(errorCode, isCustomCode)
+### error(macros, isCustomCode)
+
+Send a request to the URI provided by the VAST <Error> element. `macros` is an optional Object containing macros and their values to be used and replaced in the tracking calls.
+Pass `isCustomCode` as true in order to use any value. If false or no value is passed, the macro will be replaced using `errorCode` only if the code is a number between 100 and 999 (see <https://gist.github.com/rhumlover/5747417>). Otherwise 900 will be used.
+
+#### Parameters
+
+- **`macros : Object `** - An optional Object containing macros and their values to be used and replaced in the tracking calls.
+- **`isCustomCode: Boolean`** - Flag to allow custom values on error code.
+
+#### Example
+
+```Javascript
+const MEDIAFILE_PLAYBACK_ERROR = '405';
+
+// Bind error listener to the player
+player.on('error', () => {
+  vastTracker.error(MEDIAFILE_PLAYBACK_ERROR);
+});
+```
+
+### errorWithCode(errorCode, isCustomCode) : this method is deprecated in favor of the error method
```

Comment:
Actually, we could simplify the link to `[error method](#errormacros-iscustomcode)` IMO it's better to link the current file rather than a frozen version of a commit and more future-proof